course/assignments/reproducible-research-assignment.md

```diff
@@ -0,0 +1,21 @@
+## Reproducible Research Plan:
+
+Based on the [Data Reuse Plan](https://github.com/mozillascience/working-open-workshop/blob/gh-pages/handouts/data_reuse_plan_template.md) by Mozilla WOW.  
+
+Create a README.Rmd that describes the data or software package. It can be pretty sparse for now but you will populate it as you create your project. It must have the following sections (If a section does not apply to your project, remove it and when you submit your assignment, include a Notes.txt explaining why it is not applicable)
```

Comment:
Maybe say "must have the following sections: Project description (with data source provided); Who we are; Installation; Usage; Folders and files in project; Citation information. More detail about what to include in each section is provided below. Not all sections will be relevant to your project."